View/ViewHandler.php

```diff
@@ -448,6 +449,18 @@ private function initResponse(View $view, $format)
             if ($serializer instanceof SerializerInterface) {
                 $context = $this->getSerializationContext($view);
                 $content = $serializer->serialize($data, $format, $context);
+            //       test for symfony 2.7 version 
+            } elseif (class_exists('\Symfony\Component\Serializer\Normalizer\ObjectNormalizer') && $serializer instanceof SymfonySerializerInterface) {
+                    
+                $context = $this->getSerializationContext($view);
```

Comment:
Can you confirm that every serialization tests on FOSRest are using the JMSSerializer? What about having those same tests with the Symfony Serializer, would we have to duplicate all those too?